### PR TITLE
Add github ribbon to all pages

### PR DESCRIPTION
**Pull request by ErikHellman**
Let's add the github ribbon to all samples for easier access to the sources.

### CODE REVIEW

File: src/content/datachannel/basic/index.html [hunk 1]
```diff
@@ -66,7 +66,11 @@ <h2>Receive</h2>
     <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/datachannel/basic"
        title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
 </div>
-<a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/datachannel/basic"><img style="position: absolute; top: 0; left: 0; border: 0;" src="https://s3.amazonaws.com/github/ribbons/forkme_left_red_aa0000.png" alt="Fork me on GitHub"></a>
+
+<a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/datachannel/basic"><img
+        style="position: absolute; top: 0; left: 0; border: 0;"
```
Comment (samdutton) [inline src/content/datachannel/basic/index.html]: Would be great to move this out of the HTML.

**Review by samdutton — APPROVED** [review]
LGTM apart from request to move CSS out of HTML.

**Review by KaptenJansson — CHANGES_REQUESTED** [review]
LGTM overall but agree on not including CSS in HTML.

**Comment by ErikHellman** [comment]
I'm closing this and will open a new PR later on.

**Pull request closed without merging**